src/main.cpp

```diff
@@ -211,7 +211,7 @@ void addPackOptions(po::options_description& desc)
     po::options_description pack("Pack Options");
     pack.add_options()
         ("type,t", po::value<string>()->value_name("TYPE"),
-            "Type of the input model. Currently supported types:\n \t - caffe")
+            "Type of the input model. Currently supported types:\n \t - caffe \n \t - tensorflow")
```

Comment:
Can you refactor this to build the list dynamically?  I believe that 
```
std::vector<std::string> classification::ModelIdentifier::types();
```
Is what you need.  (See also `po::detail::cmdline::style_parser buildExtraStyleParser()`)